### DISCUSSION TITLE
fail on command: kubectl argo rollouts dashboard

### DISCUSSION THREAD

**Discussion by templarfelix** [argoproj/argo-rollouts#1122] [General]
on last binary dashboard don't exists

╰─>$ kubectl argo rollouts version

kubectl-argo-rollouts: v0.10.2+54343d8
  BuildDate: 2020-12-17T20:46:09Z
  GitCommit: 54343d8c9eb1e4b9f7e87b3da533530199916733
  GitTreeState: clean
  GoVersion: go1.13.1
  Compiler: gc
  Platform: linux/amd64

have solution?

**Comment by flerpo** [comment]
found it here...
https://github.com/argoproj/argo-rollouts/releases/v1.0.0-rc1/download/kubectl-argo-rollouts-linux-amd64
